server/utils/format.ts

```diff
@@ -2,6 +2,9 @@ const notNumber = (n: unknown): n is number => typeof n !== 'number' || Number.i
 
 export default {
   dateAndTime(date: Date): string {
+    if (typeof date === 'undefined' || date === null) {
```

Comment:
Any reason to also compare date with `undefined`, e.g. `date === undefined`?

More curious than anything. We already discussed the weirdness of `typeof null` but maybe there is some other 'JS reason' to use `typeof` here.